src/pages-conditional/eth.tsx

```diff
@@ -16,153 +24,204 @@ import InfoBanner from "../components/InfoBanner"
-  min-width: 100%;
-  margin: 0.5rem 0rem;
-  border-radius: 0px;
-`
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Flex
+    align="center"
+    justify="space-between"
+    direction={{ base: "column-reverse", md: "row" }}
+    {...props}
+  />
+)
 
-const TextDivider = styled.div`
-  margin-bottom: 2rem;
-  margin-top: 2rem;
-  width: 10%;
-  height: 1px;
-  background-color: ${(props) => props.theme.colors.searchResultBackground};
-  @media (min-width: ${(props) => props.theme.breakpoints.l}) {
-    align-self: flex-start;
-  }
-`
+const Hero = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 100%"
+    maxW="800px"
+    bgSize="cover"
+    bgRepeat="no-repeat"
+    ml={{ base: 0, md: 8 }}
+    {...props}
+  />
+)
 
-const CentralColumn = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  max-width: 960px;
-  margin: 4rem auto;
-`
+const Header = (props: ChildOnlyProp) => (
+  <Box
+    as="header"
+    flex="1 1 50%"
+    minW="300px"
+    mt={{ base: 6, md: 32 }}
+    {...props}
+  />
+)
 
-const CentralActionCard = styled(ActionCard)`
-  flex: none;
-  margin: 2rem 0;
-  .action-card-image-wrapper {
-    padding: 1rem;
-  }
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    display: flex;
+const H2 = (prop: HeadingProps) => (
+  <Heading
+    fontSize={{ base: "2xl", md: "2rem" }}
+    lineHeight={1.4}
+    mt={0}
+    {...prop}
+  />
+)
 
-    .action-card-image-wrapper {
-      min-width: 260px;
-    }
-    .action-card-content {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      margin-left: 1rem;
+const H4 = (props: ChildOnlyProp) => (
+  <Heading
+    as="h4"
+    fontSize={{ base: "md", md: "xl" }}
+    fontWeight={600}
+    lineHeight={1.4}
+    {...props}
+  />
+)
 
-      p {
-        margin-bottom: 0;
-      }
-    }
-  }
-`
+const CardContainer = (props: FlexProps) => (
+  <Flex wrap="wrap" ml={-4} mr={-4} {...props} />
+)
+
+const TokenCard = (props: ComponentProps<typeof HorizontalCard>) => (
+  <HorizontalCard minW="full" my={2} mx={0} borderRadius={0} {...props} />
+)
+
+const TextDivider = () => (
+  <Box
+    w="10%"
+    h="1px"
+    bg="searchResultBackground"
+    my={8}
+    alignSelf={{ base: "flex-start" }}
+  />
+)
+
+const CentralColumn = (props: ChildOnlyProp) => (
+  <Flex
+    direction="column"
+    align="center"
+    maxW="960px"
+    my={16}
+    mx="auto"
+    {...props}
+  />
+)
+
+const CentralActionCard = (props: ComponentProps<typeof ActionCard>) => (
+  <ActionCard
+    display={{ base: "block", sm: "flex" }}
+    flex="none"
+    my={8}
+    mx={0}
+    sx={{
+      ".action-card-image-wrapper": {
+        minW: { sm: "260px" },
+      },
+      ".action-card-content": {
+        display: { sm: "flex" },
+        justifyContent: { sm: "center" },
+        flexDirection: { sm: "column" },
+        ml: { sm: 4 },
+      },
+      p: {
+        mb: { sm: 0 },
+      },
+    }}
+    {...props}
+  />
+)
```

Comment:
Image placement on mobile res is a little low / not centred

![image](https://github.com/ethereum/ethereum-org-website/assets/62268199/1dd94f43-c407-4ec0-a787-398a83ac1e21)